cg_hermes/config/tags.py

```diff
@@ -125,6 +126,7 @@
     "cnvkit": {"description": "Tool to call copy number variations"},
     "ascatngs": {"description": "Tool to identify somatically acquired copy-number alterations"},
     "delly": {"description": "Cancer structural variant prediction tool"},
+    "svdb": {"description": "Tool to merge SV vcf files from multiple callers or individuals"},
```

Comment:
```suggestion
    "svdb": {"description": "Tool to merge SV vcf files from multiple variant callers"},
```

Also good to include `vcf2cytosure` : description: "Tool to convert SV/CNV vcf files to '.cgh' format

Reply:
It has been already defined in that file: 
https://github.com/Clinical-Genomics/hermes/blob/f971c4b018538b2486ed8edfbad3d9fbc3fadea2/cg_hermes/config/tags.py#L59

Reply:
Missed to see this part